metrics_test.go

```diff
@@ -39,6 +39,10 @@ func TestMain(m *testing.M) {
 		natsURL = nats.DefaultURL
 	}
 
+	SetErrorHandler(func(err error) {
```

Comment:
Wut? O.O

Reply:
just for test

Reply:
if we have some errors through tests we do panic